tools/mascot/src/main/java/dk/alexandra/fresco/tools/mascot/mult/MultiplySharedHelper.java

```diff
@@ -0,0 +1,32 @@
+package dk.alexandra.fresco.tools.mascot.mult;
+
+import dk.alexandra.fresco.framework.network.Network;
+import dk.alexandra.fresco.tools.mascot.MascotResourcePool;
+import dk.alexandra.fresco.tools.mascot.TwoPartyProtocol;
+import dk.alexandra.fresco.tools.ot.base.RotBatch;
+
+/**
+ * Class that stores data and resources common across the two sides of the multiplication helper
+ * protocol, {@link MultiplyLeftHelper} and {@link MultiplyLeftHelper}.
+ */
+public abstract class MultiplySharedHelper extends TwoPartyProtocol {
+
+  private final RotBatch rot;
+  private final int numLeftFactors;
```

Comment:
Please consider if `numLeftFactors` is ever being used. As far as I can see it is always set to 1, which essentially means we can get completely rid of it. It looks like this is some old stuff that we keep dragging around. The code would be a lot clearer if without this odd variable.

Reply:
... which btw. does not ever seem to be equal to the number of left factors. So at least a renaming might be good.

Reply:
Got rid of `numLeftFactors`.